about.html

```diff
@@ -13,56 +13,72 @@
 
 <body>
     <div class="container about-page">
+      <header>
+        <nav>
+                <img src="img\lambda-black.png">
+                <a href="index.html">Home</a>
+                <a href="about.html">About</a>
+                <a href="#">Products</a>
+                <a href="#">Blog</a>
+                <a href="#">Contact</a>
+            </nav>
+    </header>
+    <img class="jumbo" src="img\jumbo-about.png">
+    <div class="about">
+    <h2>About</h2>
+
+    <p>Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.</p>
+    </div>
+<div class="mid-navContainer">
```

Comment:
Make sure to keep your indentation organized for yourself and future developers who will be looking at your code